src/rutabaga_core.rs

```diff
@@ -976,6 +983,26 @@ impl Rutabaga {
         Ok(())
     }
 
+    pub fn resource_map(&mut self, resource_id: u32, addr: Option<u64>) -> RutabagaResult<()> {
```

Comment:
I think this is somewhat analogous to `vk_ext_map_memory_placed`.  So a better name would be something like:

`map_placed(&mut self, resource_id: u32, placed_addr:  u64) -> RutabagaResult<()>`